homeassistant/components/iotty/api.py

```diff
@@ -0,0 +1,49 @@
+"""API for iotty bound to Home Assistant OAuth."""
+
+import logging
+from typing import Any
+
+from aiohttp import ClientSession
+from iottycloud.cloudapi import CloudApi
+from iottycloud.device import Device
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from .const import IOTTYAPI_BASE, OAUTH2_CLIENT_ID
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.SWITCH]
+
+
+class IottyProxy(CloudApi):
+    """Provide iotty authentication tied to an OAuth2 based config entry."""
+
+    # We shall remove these
+    _devices: list[Device]
+    _entities: dict
+    _coroutine: Any
```

Comment:
Either remove these if they are not needed, or remove the comment if they are needed.

Reply:
That was indeed some code that was got lost in the transition, thanks for the heads-up!

Reply:
They are still not removed though